src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/service/CreateVideoLinkBookingService.kt

```diff
@@ -0,0 +1,11 @@
+package uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.service
+
+import org.springframework.stereotype.Service
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.model.request.VideoLinkBookingCreateRequest
+
+@Service
+class CreateVideoLinkBookingService {
```

Comment:
Was thinking of having more fine grained services instead of one service to do it all. So perhaps one for create, one for updates etc.